aws/lambda/log-classifier/ruleset.toml

```
@@ -172,6 +172,10 @@ pattern = 'Library not loaded: .*$'
 name = 'Bad response status code'
 pattern = '^##\[error\]Response status code does not indicate success: .*$'
 
+[[rule]]
+name = 'Python Test timeout (KeyboardInterrupt)'
+pattern = '\!+ KeyboardInterrupt \!+'
```

```suggestion
pattern = '!+ KeyboardInterrupt !+'
```

The log classifier is coded in Rust, so it looks like you don't need to escape `!`.  I test it out on https://rustexp.lpil.uk/